controllers/imagerepository_controller.go

```diff
@@ -230,6 +234,139 @@ func (r *ImageRepositoryReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	return ctrl.Result{}, nil
 }
 
+func (r *ImageRepositoryReconciler) NotificationExistsInQuayByUUID(UUID string, imageRepository *imagerepositoryv1alpha1.ImageRepository) (quay.Notification, error) {
```

Comment:
Probably it exists a more elegant way to have this two functions (or not). Let me know any idea.